examples/pydantic-ai/agent.py

```
@@ -0,0 +1,81 @@
+        result = await agent.run(query)
+        return result.output
+    except Exception as e:
+        return WeatherResponse(
+            location="Unknown", temperature="N/A", description=f"Error: {str(e)}"
+        )
+
+
+if __name__ == "__main__":
+    import asyncio
+
+    async def test_agent():
```

Ruff (0.11.9)

Missing return type annotation for public function `test_agent`

Add return type annotation: `None`

(ANN201)